py/private/py_venv.bzl

```diff
@@ -1,4 +1,10 @@
-"""Implementation for the py_binary and py_test rules."""
+"""Create a Python virtualenv directory structure.
+
+Uses https://github.com/prefix-dev/rip.
```

Comment:
Do we need to mention this? Seems like an implementation detail that's hidden from the end user.

Reply:
We don't have to, but I think it's very interesting. I'll make it clear that you cannot depend on it.